README.md

```diff
@@ -1,3 +1,10 @@
 # GrowBox
-## Установка:
-Инструкция по установке и настройке приложения GrowBox находится на http://wiki.amperka.ru/hq  в разделе **Операционная деятельность/Образы**.
+GrowBox is a hydroponic plant growing system.
+
+This repository contains the source code of Raspberry Pi and Arduino Leonardo, which connects a set of sensors and actuators to maintain the microclimate in the growbox.
+## License:
```

Comment:
- Empty lines around the heading are missing
- No punctuation at the end of a heading.